webrtc/videoroom/lib/videoroom_web/endpoint.ex

```diff
@@ -0,0 +1,35 @@
+defmodule VideoRoomWeb.Endpoint do
+  use Phoenix.Endpoint, otp_app: :membrane_videoroom_demo
+
+  socket("/socket", VideoRoomWeb.UserSocket,
+    websocket: true,
+    longpoll: false
+  )
+
+  plug(Plug.Static,
+    at: "/",
```

Comment:
```suggestion
    at: "/static",
```

Reply:
It happens that the plug serves `/static` well and `/` does not work.